### PR TITLE
Fix minor doc issue, update to enable apis.

### PR DESCRIPTION
**Pull request by n3wscott**
fixes: https://github.com/google/knative-gcp/issues/243

Also update docs to enable APIs used in the resources.

### CODE REVIEW

File: docs/pubsub/README.md [hunk 1]
```diff
@@ -12,7 +12,8 @@ The following are prerequisites for `Channel`, `Topic` and `PullSubscription`.
    project id, and also set your project ID as default using
    `gcloud config set project $PROJECT_ID`.
 
-1. Enable the `Cloud Pub/Sub API` on your project:
+1. [Enable](https://cloud.google.com/endpoints/docs/openapi/enable-api) the
```
Comment (vaikas-google) [inline docs/pubsub/README.md]: Why do all these links point to 'Cloud Endpoints'? Is there no high level page that's not product specific?

Comment (n3wscott) [reply docs/pubsub/README.md]:
```Cloud Endpoints
Develop, deploy, and manage APIs on any Google Cloud back end.```

I think Cloud Endpoints is the cute name for the API auth manager?

Comment (n3wscott) [reply docs/pubsub/README.md]: There is nothing product specific on this page, it is the generic how to page for enabling apis

Comment (vaikas-google) [reply docs/pubsub/README.md]: yes it is, it's under Cloud Endpoints:
https://cloud.google.com/endpoints/

**Pull request merged**